utils/translation_service.py

```
@@ -0,0 +1,251 @@
+                if value and all(isinstance(item, str) for item in value):
+                    result[key] = [await self.translate_text(item, target_lang) for item in value]
+        
+        return result
+
+
+# 全局翻译服务实例（延迟初始化）
+_translation_service_instance: Optional[TranslationService] = None
+
+
+def get_translation_service(config_manager) -> TranslationService:
+    """获取翻译服务实例（单例模式）"""
```

Ruff (0.14.8)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)